packages/cli/src/commands/prisma.js

```diff
@@ -0,0 +1,73 @@
+import fs from 'fs'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from '@redwoodjs/internal'
+
+import c from 'src/lib/colors'
+
+export const command = 'prisma [...commands]'
+export const description = 'Run Prisma CLI with built-in helpers'
+
+export const builder = (yargs) => {
+  const argv = process.argv.slice(3)
+  const paths = getPaths()
+
+  // descriptions below generate list when `rw prisma --help` is run
+  yargs
+    .example("'yarn rw prisma migrate dev'", 'No additional options required')
+    .positional('db push', {
+      alias: 'push',
+      description:
+        'Push the state from your Prisma schema to your database without creating a migration',
+    })
+    .positional('db seed', {
+      description: "Seed your database. Requires a 'seed.js' file",
+    })
+    .positional('migrate dev', {
+      description:
+        'Create a migration from schema.prisma; apply it to the dev database',
+    })
+    .positional('migrate reset', {
```

Comment:
As the reset command is destructive -- can it provide a warning and confirm? Or is the idea to let the prisma cli do that?

Reply:
The Prisma CLI does handle warning + interactive confirmation.
<img width="410" alt="Screen Shot 2021-02-05 at 8 26 06 AM" src="https://user-images.githubusercontent.com/2951/107060421-cc432680-678b-11eb-970e-32abd739558e.png">